docs/markdown/Python/python-goals/python-check-goal.md

```diff
@@ -130,25 +133,35 @@ the stubs are only used when running MyPy and are not included when, for example
 ```toml pants.toml
 [mypy]
 extra_type_stubs = ["types-requests==2.25.12"]
-# Set this to a path, then run `pants generate-lockfiles --resolve=mypy-extra-type-stubs`. 
+# Set this to a path, then run `pants generate-lockfiles --resolve=mypy-extra-type-stubs`.
 extra_type_stubs_lockfile = "3rdparty/python/mypy_extra_type_stubs.lock
 ```
 
 ### Add a third-party plugin
 
-Add the plugin to the `extra_requirements` option in the `[mypy]` scope, then update your `mypy.ini` to load the plugin:
+Add the plugin to the `requirements` option in the `[mypy]` scope, and create a `requirements.txt` file for the version. Then specify the resolve that the requirements should be installed from:
 
 ```toml pants.toml
+[python.resolves]
+mypy = "3rdparty/python/mypy-lock.txt"
+
 [mypy]
-extra_requirements.add = ["pydantic==1.6.1"]
+requirements.add = ["pydantic"]
```

Comment:
This is no longer necessary! Now if `requirements` is empty the entire lockfile is used. So it may be easier to leave off entirely here (the main lockfiles page explains when you do want to use `requirements`).